themes/mCommerceTheme/styles/components/LviReview.json

```diff
@@ -0,0 +1,7 @@
+{
+	".lviReview": {
+		"height": 150,
+		"backgroundColor": "${backgroundMain}",
```

Comment:
remove backgroundColor if background on the page is same as this

Reply:
It is necessary because of ios.